features/admin.users.v1/pages/users.tsx

```diff
@@ -672,6 +724,33 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
                 } }
                 isLoading={ isUserListFetchRequestLoading }
             >
+                { selectedAccountStatusFilters.length > 0 && (
+                    <Label.Group>
+                        { selectedAccountStatusFilters.map((accountStatusFilterKey: string) => {
+                            const statusFilterOption: AccountStatusFilterOption =
+                            USER_ACCOUNT_STATUS_FILTER_OPTIONS.find(
+                                (option: AccountStatusFilterOption) => option.key === accountStatusFilterKey);
+
+                            if (!statusFilterOption) {
+                                return null;
+                            }
+
+                            return (
+                                <Label
```

Comment:
we may be able to use the Chip component from oxygen UI for this

Reply:
I initially used the chip component and then to align with connections page tags, I changed the code to use label component